### DISCUSSION TITLE
API seems to not be working very well

### DISCUSSION THREAD

**Discussion by iainmck29** [perplexityai/api-discussion#35] [General]
Currently trying to do some experimental work with the API. It was returning some reasonably good results but for a while now it's been throwing a lot of 500 errors. Moreover, I recently got a full blown HTML error response page which is very strange behaviour for an API.

Edit: I'm also receiving rate limiting errors despite having purposefully limited my testing to do no more than 5/min to stay well under the rate limits. No idea why this is being returned.

What's worse is that I seem to be being charged for the API to only send back errors!

Do you have plans to improve the API?

I'd be happy to share some of my queries/responses for sake of improvement.

**Comment by rafs2006** [comment]
Hey @iainmck29! Could you please share some of the queries you are sending?
